09-student-guide.Rmd

```diff
@@ -15,10 +15,11 @@ Make sure you have an AnVIL account (refer to https://jhudatascience.org/AnVIL_B
 This activity will teach you how to use the AnVIL platform to:
 
 1. Get started working on AnVIL
-1. Launch the Galaxy tool
-1. Examine fastq sequence data files
-1. Align sequence data to a reference genome
-1. View the aligned data and reference genomes interactively
+2. Launch the Galaxy tool
```

Comment:
The numbers get rendered automatically by markdown 😄

Reply:
Bah, that's annoying! I hope I didn't screw the numbering up again.